src/preprocessing/extraction.py

```diff
@@ -172,7 +174,16 @@ def convert_and_save_with_page_numbers(input_file_path, output_file_path):
         print(f"Error: Input file not found at {input_file_path}", file=sys.stderr)
         return
 
-    converter = DocumentConverter()
+    # Disable OCR and table structure extraction for faster processing
+    pipeline_options = PdfPipelineOptions()
+    pipeline_options.do_ocr = False
+    pipeline_options.do_table_structure = False
```

Comment:
I think we should keep table structures because we're dealing with a DB textbook.

In some cases, it might provide good context to the written text:
<img width="424" height="295" alt="Image" src="https://github.com/user-attachments/assets/04d2c25d-d675-4dee-91ab-d8e30280f42d" />